tls/s2n_client_extensions.c

```diff
@@ -404,3 +417,9 @@ static int s2n_recv_client_renegotiation_info(struct s2n_connection *conn, struc
     conn->secure_renegotiation = 1;
     return 0;
 }
+
+static int s2n_recv_client_sct_list(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
Not sure if you need me to do anything here or not? I added a comment noting that we're skipping it.